src/paratii.js

```diff
@@ -80,6 +85,87 @@ class Paratii extends ParatiiCore {
   getAccount () {
     return this.eth.getAccount()
   }
+  /**
```

Comment:
It would probably be nicer to spread the checks over the different submodules, so the code stays modular. I.e. the check for the db in `paratii.db.js`, the checks for ipfs in the ipfs submodule, etc. We can then run all the diagnose command together there.

Reply:
Another point (more an idea): instead of having these checks just write some lines to a log, we could introduce a new function (`checkServices`) that returns an object with the various things it checks together with response times, i.e. something like:

```
{
   'db.provider': { // <- this is the name of the var passed in the config
        'url': 'url of provider',
        'response': '200 OK',
        'responseTime': 300, // in ms
  }
}
```

Reply:
I moved the checks to their own submodules.

Should I take on implementing the checkServices functionality?

Reply:
would appreciate that, yes :-)

Reply:
I implemented checkServices :)